priv/perf/config/test.exs

```diff
@@ -1,6 +1,25 @@
 use Mix.Config
 
-config :load_test,
+config :logger, level: :info
```

Comment:
Not sure the intention here. 

~If it is Tesla being too annoying, probably this:
https://hexdocs.pm/tesla/Tesla.Middleware.Logger.html#module-logger-debug-output~

[edit: sigh....I tried and it is not working 😢]

Reply:
I really do not like `tesla`'s spam, and indeed the config does not work. I can set it back to `debug` if you say want.

Reply:
nah, it is fine. I understand it to be annoying.